package challenges.concurrentexecution;

public class FailException extends RuntimeException
{
    public FailException()
    {
        super();
    }
}
